src/WakaTime.Shared.ExtensionUtils/Heartbeat.cs

```diff
@@ -6,5 +6,43 @@ public class Heartbeat
         public string Timestamp { get; set; }
         public string Project { get; set; }
         public bool IsWrite { get; set; }
+        public HeartbeatCategory Category { get; set; }
+        public EntityType EntityType { get; set; }
+    }
+
+    /* Matheus Rocha (MattLebrao) on Aug 31 2020 @ 19h28:
```

Comment:
I do prefer to not have any comment in the code since it's well documented in the client's repository.